usage-collection/src/main/proto/usage.proto

```diff
@@ -1058,6 +1058,8 @@ message CDPOperationDetails {
     string flowState = 9;
     // Correlation ID to correlate events written together
     string correlationId = 10;
+    // Cloud provider identifier
+    CDPEnvironmentsEnvironmentType.Value environmentType = 11;
```

Comment:
the type is the provider info here?

Reply:
yes, it's the same field as in CDPEnvironmentDetails